### ISSUE TITLE
Plugin install button stays the same after installing it

### ISSUE THREAD

**Issue by paulopmt1** [Automattic/wp-calypso#97142]
### Quick summary

The plugin's interface doesn't detect when a plugin is installed on Atomic sites.

### Steps to reproduce

1. Open an Atomic site
2. Go to https://wordpress.com/plugins/download-manager/{siteDomain}
3. Install it
4. While installing, go back to the step 2
5. Refresh the page and check that the CTA remains the same

### What you expected to happen

The main CTA remains `Install and activate` after the plugin installation

### What actually happened

The CTA should offer a plugin management instead

### Impact

Some (< 50%)

### Available workarounds?

Yes, easy to implement

**Comment by Robertght** [comment]
I'm unable to replicate this on my end.

This is what I see when I got back to the previous screen while the plugin is instilling:

![Image](https://github.com/user-attachments/assets/f779cbe1-ad73-4c25-b3f9-2ee5a83c4e6f)

I can confirm the behavior is consistent proxied/non-proxied and while using Chrome.

@paulopmt1 could you give this another spin?

**Comment by paulopmt1** [comment]
Yeah, I tried to simulate it again and couldn't. It looks like it was something odd at that time. Thanks for testing it @Robertght!
